player/mpv.go

```diff
@@ -175,6 +176,18 @@ func (mpv *MPV) Pause() {
 	mpv.setProperty("pause", "yes")
 }
 
+func (mpv *MPV) ToggleMute() {
+	mute := "no"
+	if mpv.mute {
+		mute = "yes"
+	}
+	mpv.mute = !mpv.mute
```

Comment:
Condition is reversed without this change. You want to change the value before using it to set the property.

```suggestion
	mpv.mute = !mpv.mute
	mute := "no"
	if mpv.mute {
		mute = "yes"
	}
```

Reply:
done